contentcuration/contentcuration/views/admin.py

```diff
@@ -141,19 +141,11 @@ def get_channel_kind_count(request, channel_id):
         raise SuspiciousOperation("You are not authorized to access this endpoint")
 
     channel = Channel.objects.get(pk=channel_id)
-
-    sizes = ContentNode.objects\
-        .prefetch_related('assessment_items')\
-        .prefetch_related('files')\
-        .prefetch_related('children')\
-        .filter(tree_id=channel.main_tree.tree_id)\
-        .values('files__checksum', 'assessment_items__files__checksum', 'files__file_size', 'assessment_items__files__file_size')\
-        .distinct()\
-        .aggregate(resource_size=Sum('files__file_size'), assessment_size=Sum('assessment_items__files__file_size'))
+    data = get_node_details(channel.main_tree)
```

Comment:
Hmm this should be in a utility module I believe. Maybe utils/nodes.py?

Reply:
Let's address this when we start refactoring the code